Example/SourcePointMetaAppUITests/Helpers/MetaApp.swift

```diff
@@ -399,11 +400,11 @@ extension MetaApp: GDPRUI {
     }
 
     var PersonalisedAdsSwitch: XCUIElement {
-        consentUI.switches["Create a personalised ads profile"].firstMatch
+        consentUI.switches.element(boundBy: 2) // == "Create a personalised ads profile"
```

Comment:
@Nevazhnovu why did you opt for changing these lines? to make it more robust to changes on the text? If that's the case, I think switching from 
```
consentUI.switches["Create a personalised ads profile"].firstMatch
```
to 
```
consentUI.switches.element(boundBy: 2)
```
makes it less readable (though more robust). I'd still go with the more readable approach.

Reply:
@andresilveirah it somehow doesn't detect the element if search by it's name but works perfectly fine with index

Reply:
Hm... and the text is definitely definitely there?

Reply:
@andresilveirah  120% sure